packages/mobile-app-did/js/pages/Discover/components/SubPages/Earn/index.tsx

```diff
@@ -8,14 +8,17 @@ import { TextL, TextS } from 'components/CommonText';
 import NoData from 'components/NoData';
 import { useDiscoverJumpWithNetWork } from 'hooks/discover';
 import React, { useCallback } from 'react';
-import { View, StyleSheet, Image, TouchableOpacity } from 'react-native';
+import { View, StyleSheet, ScrollView, Image, TouchableOpacity } from 'react-native';
 import { pTd } from 'utils/unit';
 
 export const EarnPage = () => {
   const { earnList = [] } = useDiscoverData();
   return (
     <View style={styles.container}>
-      {earnList.length > 0 ? earnList.map((item, index) => <EarnItem key={index} {...item} />) : <NoData />}
+      <ScrollView showsVerticalScrollIndicator={false}>
+        {earnList.length > 0 ? earnList.map((item, index) => <EarnItem key={index} {...item} />) : <NoData />}
+        <View style={styles.gap} />
+      </ScrollView>
```

Comment:
`{earnList.length > 0 ? (
        <ScrollView showsVerticalScrollIndicator={false}>
          {earnList.map((item, index) => (
            <EarnItem key={index} {...item} />
          ))}
          <View style={styles.gap} />
        </ScrollView>
      ) : (
        <NoData />
      )}`
maybe this is better